src/assets/cards/mastercard.tsx

```diff
@@ -7,8 +7,8 @@ const MasterCard: FC = () => {
 
   return (
     <svg
-      width="24px"
-      height="16px"
+      width="24"
```

Comment:
I'm not sure why the px unit should be removed

Reply:
It seems to works both ways, but it's more consistent with other svg that does not use the unit :)